common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -129,7 +130,7 @@ object CuratedRecipe {
       ),
       published = Some(
         ChangeRecord(
-          date = r.publicationDate.toInstant.toEpochMilli,
+          date = DateTime.now.toInstant.getMillis,
```

Comment:
@mchv  @annebyrne Don't we want the publication date to be that of when the original recipe article was created?